src/ansible_navigator/runner/command_async.py

```diff
@@ -7,6 +7,7 @@
 """
 
 from queue import Queue
+
 from ansible_runner import run_command_async  # type: ignore[import]
```

Comment:
It'd be nice to have a section declared in the isort config that would list the Ansible-land dependencies. As in things from our ecosystem/colleagues. Separately from the rest of the third-party libs.

Reply:
@webknjaz  I'm not sure what you change/action you are suggesting here..... can you elaborate?

Reply:
resolving conversation, please reopen if the concerns and question are unanswered, ty for the feedback. I am curious about this suggestion, let's continue the idea as an issue/feature request in the future.

Reply:
https://github.com/ansible/ansible-navigator/issues/781